interview-questions.md

```diff
@@ -4,39 +4,39 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. Why would you use the method super()?
 
-  Your answer:
+  Your answer:you would use it to call the base classe's constructor
 
   Researched answer:
 
 
 
 2. What is a virtual DOM?
 
-  Your answer:
+  Your answer:its the representation of the UI
 
   Researched answer:
 
 
 
 3. When creating a basic (stateless) class component in React, what are the necessary elements needed to display "Hello World" in the browser?
 
-  Your answer:
+  Your answer: const = props => React.Element
```

Comment:
Not really sure what you're saying here. If i was asking this question as someone interviewing you, I'd want you to elaborate more. If you're unsure of the answer, I think it would be acceptable to respond something like "I'm not sure, but I would look into the react documentation to find out."